hosts_waku.tf

```diff
@@ -0,0 +1,35 @@
+module "hosts" {
+  source = "github.com/status-im/infra-tf-multi-provider"
+
+  /* node type */
+  group  = "waku"
+  env    = "waku"
```

Comment:
We need to reuse already existing `hosts.tf` file and make this `env` and `group` values conditional.

And we cannot depend on `workspace` to be `test` because there's already a `test` workspace:
```
 > t workspace list       
  default
  prod
* test
```